packages/common/src/utilities/Jupyter/index.ts

```diff
@@ -76,8 +76,14 @@ interface JupyterWebSocketMessageInputReplyContent {
   value: string;
 }
 
+interface JupyterWebSocketMessageStreamResultContent {
+  name: string;
+  text: string;
+}
+
 enum JupyterWebSocketMessageType {
```

Comment:
nit: why the use of enum instead of `type JupyterWebSocketMessageType = "execute_result" | "stream"`?

Reply:
That's a fair point.  The file was actually written by Shaofeng, adopted from another project.  In fact, at the top of the file, you see:

```
/* TODO: for now, copy-pasted this file from an external source -- so disable tslint temporarily */
/* tslint:disable */
```

But, adding @shaofengzhu for his awareness for the future.